src/Formidable.res

```diff
@@ -392,18 +392,20 @@ module Make = (
       let (isFocused, setIsFocused) = React.useState(() => false)
 
       let validationNames =
-        props["validations"]->Option.mapWithDefault(list{}, validations =>
-          validations->ListExtra.flatMap(Validations.getName)
+        props["validations"]->Option.mapWithDefault([], validations =>
+          validations->ArrayExtra.flatMap(validation =>
+            validation->Validations.getNames->List.toArray
```

Comment:
Tried to isolate the conversion code as much as I could.
When ready, we can simply drop the `List.toArray` function call and use arrays instead of lists